blob/blob.go

```diff
@@ -420,6 +421,9 @@ func (b *Bucket) As(i interface{}) bool {
 // ErrorAs converts i to provider-specific types.
 // See Bucket.As for more details.
```

Comment:
Add the comment here about it panicking if i is nil or not a pointer.

Reply:
Done.